Cython/Utility/FunctionArguments.c

```diff
@@ -612,15 +624,15 @@ static CYTHON_INLINE __Pyx_FastcallTuple_obj __Pyx_FastcallTuple_SliceIndex(__Py
     Py_ssize_t out_len = stop - start;
     if (out_len < 0) out_len = 0;
 
-#if CYTHON_COMPILING_IN_CPYTHON
+#if CYTHON_METH_FASTCALL
 #if CYTHON_VECTORCALL
     if ((start > 0) || (in.nargs & PY_VECTORCALL_ARGUMENTS_OFFSET)) {
         out_len |= PY_VECTORCALL_ARGUMENTS_OFFSET;
     }
-#endif // CYTHON_VECTORCALL
+#endif /* CYTHON_VECTORCALL */
```

Comment:
Yeah, you probably just noticed here that `//` comments only get discarded at the beginning of the line, not in arbitrary later places (and `/* … */` only if it's within one line). That's (half) intentional. Most comments should normally get discarded, but a few are intended to stay in.